coforma/swift-tech-challenge
# Issue/Hero Image

# Title
Issue/Hero Image

# Summary
This PR adds a hero image to the top of the browse page

Fixes #34

## Desktop

![Screenshot 2024-03-14 at 10 36 58 AM](https://github.com/coforma/swift-tech-challenge/assets/19439679/4a00b209-8114-400f-8d6a-2e6f863fec67)


## Mobile

![Screenshot 2024-03-14 at 10 41 19 AM](https://github.com/coforma/swift-tech-challenge/assets/19439679/9ca6c45e-e310-48ad-8f3d-f3a9e78f88b6)

## Type of change


# How To Test

1. run `yarn dev`, then navigate to `localhost:3000`
2. See the new mock hero image in action!

# Checklist:
